sdk/servicebus/azure-servicebus/samples/async_samples/deadletter_queue_async.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show receiving dead-lettered messages from a Service Bus Queue asynchronously.
```

Comment:
counterpart in sync?

Reply:
I assumed since this was a standalone sample that's needed for docs + re-directing when looking at the .NET sample, it wouldn't require a counterpart. But will add 👍